ext/Shared/__init__.lua

```diff
@@ -1,51 +1,59 @@
-GameObjectOriginType = {
-	Vanilla = 1,
-	Custom = 2,
-	CustomChild = 3
+---@class CustomLevelLoaderConfig
+---@field USE_HTTP boolean
+---@field HTTP_ROOT string
+local Config = require "__shared/Config"
+
+local GameObjectOriginType = {
+ Vanilla = 1,
+ Custom = 2,
+ CustomChild = 3
 }
 
-local CLIENT_TIMEOUT = 25
+local CLIENT_TIMEOUT = 25.0
 local SP_TERRAIN_WORLD_PART_DATA_GUID = Guid('68D438B3-FF1B-47D7-BCB4-F484E67CA700')
 local SP_TERRAIN_WORLD_PART_REFERENCE_OBJECT_DATA_GUID = Guid('93842B6D-0185-483D-9EF5-AD2B47BDABDE')
 
 -- This is a global table that stores the save file data as a Lua table. Will be populated on-demand by
 -- the server via NetEvents on the client-side
 -- Stores LevelData DataContainer guids.
-local m_PrimaryLevelGuids = nil
+local m_PrimaryLevelGuids = {}
 
 local m_IndexCount = 0
-local m_OriginalLevelIndeces = {}
 local m_LastLoadedLevelName = nil
 local m_LastLoadedGameModeName = nil
 local m_ObjectVariations = {}
 local m_PendingVariations = {}
-local m_PrimaryLevelGuids = { }
-local m_CustomLevelData = { }
+local m_CustomLevelData = {}
 
 local function PatchOriginalObject(p_Object, p_World)
 	if p_Object.originalRef == nil then
 		print("Object without original reference found, dynamic object?")
 		return
 	end
+
 	local s_Reference = nil
+
 	if p_Object.originalRef.partitionGuid == nil or p_Object.originalRef.partitionGuid == "nil" then -- perform a search without partitionguid
-		 s_Reference = ResourceManager:SearchForInstanceByGuid(Guid(p_Object.originalRef.instanceGuid))
-		 if s_Reference == nil then
-		 	print("Unable to find original reference: " .. p_Object.originalRef.instanceGuid)
-		 	return
-		 end
+		s_Reference = ResourceManager:SearchForInstanceByGuid(Guid(p_Object.originalRef.instanceGuid))
+		if s_Reference == nil then
```

Comment:
newline before and after an if statement, unless at the beginning or end of a scope